docs/conf.py

```diff
@@ -121,17 +121,15 @@ def _indicator_table(module):
 templates_path = ["_templates"]
 
 # The suffix(es) of source filenames.
-# You can specify multiple suffix as a list of string:
-#
-# source_suffix = ['.rst', '.md']
-source_suffix = [".rst", ".ipynb"]
+# If a list of string, all suffixes will be understood as restructured text variants.
+source_suffix = [".rst"]
 
 # The master toctree document.
 master_doc = "index"
 
 # General information about the project.
 project = "xclim"
-copyright = "2018, Ouranos Inc., Travis Logan, and contributors"
+copyright = "2021, Ouranos Inc., Travis Logan, and contributors"
```

Comment:
Set it and forget it?
```suggestion
copyright = f"2018-{datetime.datetime.now().year}, Ouranos Inc., Travis Logan, and contributors"
```

Reply:
Woops! This is going to need an import of `datetime`! My bad!